zero_to_hero_guide/quickstart.md

```diff
@@ -1,6 +1,26 @@
+# Quickstart Guide
```

Comment:
let's move this under docs/ please and let's add a prominent pointer to the README. Even at the top-level breadcrumbs right now would be fine with me.